src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json

```
@@ -94,7 +94,7 @@
       "placeholderMaxPurchase": "Maximalabnahme ...",
       "placeholderPurchaseSteps": "Verkaufsintervall (Stückzahl) ...",
       "placeholderRestockTime": "Wiederauffüllintervall ...",
-      "placeholderEan": "Gib die EAN-Nummer ein ...",
+      "placeholderEan": "Gib die GTIN-Nummer ein ...",
```

```suggestion
      "placeholderEan": "Gib die GTIN ein ...",
```
GTIN stays for `Global Trade Item **Number**`

---

Well.. good point! I changed it.